plugin/src/server/sc/plugin2021/GamePlugin.kt

```diff
@@ -28,15 +31,16 @@ class GamePlugin: IGamePlugin {
                     Color::class.java, Team::class.java)
         
         fun registerXStream() {
```

Comment:
why don't you merge registerXStream and loadXStream?
They do the exact same thing except one returns the xstream config, right?

Reply:
yes, it's separated to make the intention clear